src/formatDate.js

```diff
@@ -8,7 +8,31 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateObject = {};
+  const separator = fromFormat[3];
+  const newSeparator = toFormat[3];
+  const splitedDate = date.split(separator);
```

Comment:
The variable name 'splitedDate' is a typo. The correct spelling is 'splitDate'. It's important to use proper spelling for variable names to maintain code readability and professionalism.